test/ranges-odr-test.cc

```diff
@@ -0,0 +1,22 @@
+// Formatting library for C++ - the core API
+//
+// Copyright (c) 2012 - present, Victor Zverovich
+// All rights reserved.
+//
+// For the license information refer to format.h.
+//
+// Copyright (c) 2018 - present, Remotion (Igor Schulz)
+// All Rights Reserved
+// {fmt} support for ranges, containers and types tuple interface.
```

Comment:
This part only applies to ranges.h, please remove.

Reply:
Removed, and explained why this content.